Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5308,6 +5309,39 @@ def com_google_fonts_check_stylisticset_description(ttFont):
         yield PASS, "OK"
 
 
+@check(
+    id="com.google.fonts/check/os2/fsselectionbit7",
```

Comment:
let's call this **com.google.fonts/check/os2/use_typo_metrics**

Reply:
https://github.com/googlefonts/fontbakery/pull/3314/commits/af26f3a114b3e97f73b36bee4eced3cc71197410